Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient.m

```diff
@@ -471,7 +487,13 @@ - (NSURLSessionDataTask *)completeCheckout:(BUYCheckout *)checkout withApplePayT
 	}
 	else {
 		NSString *tokenString = [[NSString alloc] initWithData:token.paymentData encoding:NSUTF8StringEncoding];
-		NSDictionary *paymentJson = @{ @"payment_token" : @{ @"payment_data" : tokenString, @"type" : @"apple_pay" }};
+		NSDictionary *paymentJson = @{
+									  @"payment" : @{
```

Comment:
Not sure I understand as we don't have a model for this JSON structure. Its just simple boilerplate, as it was before, just different structure.